src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -52,45 +51,212 @@ import { LOG } from '../constants/logLevel';
+        /**
+         * Reference to the `window` object
+         *
+         * @property $window
+         * @type {JQuery|HTML Element}
+         */
         this.$window = $(window);
+
+        /**
+         * Reference to the `#canvases` tag which acts as the container
+         * element for all the `<canvas />` elements
+         *
+         * @property $element
+         * @type $element {JQuery|HTML Element}
+         * @default $element
+         */
         this.$element = $element;
 
-        this._scopeModel = scopeModel;
+        /**
+         * @property _aircraftController
+         * @type {AircraftController}
+         * @private
+         */
+        this._aircraftController = aircraftController;
+
+        /**
+         * @property _navigationLibrary
+         * @type {NavigationLibrary}
+         * @private
+         */
         this._navigationLibrary = navigationLibrary;
+
+        /**
+         * @property _scopeModel
+         * @type {ScopeModel}
+         * @private
+         */
+        this._scopeModel = scopeModel;
+
+        /**
+         * @property _eventBus
+         * @type {EventBus}
+         * @private
+         */
         this._eventBus = EventBus;
 
         prop.canvas = canvas;
         this.canvas = canvas;
-        this.canvas.contexts = {};
+
+        /**
+         * @property _context
+         * @type {object<string, HTMLCanvasContext>}
+         * @private
+         */
+        this._context = {};
+
+        // TODO: this property will be moving out of the `CanvasController` under issue:
+        // [#726](https://github.com/openscope/openscope/issues/726)
+        /**
+         * `y` axis pan value
+         *
+         * @property panY
+         * @type {number}
+         * @default 0
+         */
         this.canvas.panY = 0;
+
+        // TODO: this property will be moving out of the `CanvasController` under issue:
+        // [#726](https://github.com/openscope/openscope/issues/726)
+        /**
+         * @property panX
+         * @type {number}
+         * @default 0
+         */
         this.canvas.panX = 0;
-        // resize canvas to fit window?
-        this.canvas.resize = true;
-        // all canvases are the same size
+
+        /**
+         * Flag used to determine if the canvas dimensions should be resized
+         *
+         * @property _shouldResize
+         * @type {boolean}
+         * @default true
+         * @private
+         */
+        this._shouldResize = true;
+
+        // TODO: this property will be moving out of the `CanvasController` under issue:
+        // [#726](https://github.com/openscope/openscope/issues/726)
+        /**
+         * Enum used for storing canvas size values
+         *
+         * @property size
+         * @type {object}
+         */
         this.canvas.size = {
             height: DEFAULT_CANVAS_SIZE.HEIGHT,
             width: DEFAULT_CANVAS_SIZE.WIDTH
         };
```

Comment:
Why not just `this.canvas.height` and `this.canvas.width`?

Reply:
I agree!  I found it like this. This will be going away under #726